fastlane/Fastfile.swift

```diff
@@ -150,10 +150,6 @@ class Fastfile: LaneFile {
         desc("Build and Test project")
         Test.buildAndTest(
             environment: .staging,
-            targets: [
-                Constant.testTarget,
-                Constant.kifUITestTarget
```

Comment:
Do we still need these 2 Constants.?

Reply:
@blyscuit, I've removed those constants. We don't need them, since all the tests will be run (base on the configuration of scheme)